Docs/upgrade.md

```diff
@@ -1,8 +1,38 @@
 # Upgrade LoRaWAN to a new version
 
-## Release 1.0.0
+## Release 1.0.1
 
-We recommend re-deploying your solution based on the 1.0.0 release if you have been working with a pre-release version. If you prefer to update your existing installation, the following lists the steps required.
+We recommend re-deploying your solution based on the 1.0.1 release if you have been working with a pre-release version. If you prefer to update your existing installation, the following lists the steps required.
```

Comment:
Rephrase 
"If you prefer to update your existing installation, the following lists the steps required." 

to something like:

"If you prefer to update your existing installation, the following lists describes the required steps."